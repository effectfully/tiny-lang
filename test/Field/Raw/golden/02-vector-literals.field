let #one   = { T };
let #two   = { T , F };
let #three = { T , F , T };
let #four  = { T , F , T , F };