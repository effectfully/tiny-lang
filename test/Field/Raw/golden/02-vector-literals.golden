Program [] Statements [ELet (Var "one",Some Vector) (EConst (Some (UniConst Vector [True]))),ELet (Var "two",Some Vector) (EConst (Some (UniConst Vector [True,False]))),ELet (Var "three",Some Vector) (EConst (Some (UniConst Vector [True,False,True]))),ELet (Var "four",Some Vector) (EConst (Some (UniConst Vector [True,False,True,False])))]